src/components/SelectedPokemon/SelectedPokemon.tsx

```diff
@@ -1,15 +1,49 @@
 import * as React from 'react';
-import { Wrapper, PokemonCardContent, PokemonImageWrapper, PokemonName, PokemonSkills } from './SelectedPokemon.s';
+import { connect } from 'react-redux';
+import { addToFavorites } from '../../actions';
+import { Wrapper, PokemonCardContent, PokemonImageWrapper, PokemonName, PokemonSkills, PokemonStarWrapper, Image } from './SelectedPokemon.s';
 import { Pokemon }  from '../../model/Pokemon';
 
-type Props = {
-  selectedPokemon: Pokemon
+type StateProps = {
+  selectedPokemon: Pokemon,
+  favorites: Array<Pokemon>
+
+}
+
+type DispatchProps = {
+  addToFavorites: (obj: object) => {
```

Comment:
tutaj nie musisz, aż tak 'głęboko' typować akcji, wystarczy 
```
addToFavorites: (obj: object) => void
```

zresztą chyba to co napisałeś nie jest poprawne bo ta funkcja po dispatchu nie zwróci takiego obiektu